### PR TITLE
feat: make images to build sources on start

### PR DESCRIPTION
**Pull request by mudler**
Fixes: #102

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -79,15 +79,24 @@ go-gpt2/libgpt2.a: go-gpt2
...
+prepare-sources: go-llama go-gpt2 go-gpt4all-j
+
+rebuild:
+	$(MAKE) -C go-llama clean
+	$(MAKE) -C go-gpt4all-j clean
+	$(MAKE) -C go-gpt2 clean
+	$(MAKE) build
+
+prepare: prepare-sources go-llama/libbinding.a go-gpt4all-j/libgptj.a go-gpt2/libgpt2.a replace
+	$(GOCMD) mod download
```
Comment (mudler) [inline Makefile:99]: argh, that should go in prepare-sources

**Comment by LarsBingBong** [comment]
I guess I'll wait with using this branch?

**Comment by mudler** [comment]
> I guess I'll wait with using this branch?

Now should be good to test ! 👍

**Comment by LarsBingBong** [comment]
I've now tried this ... still unfortunately no response is returned from the model .. "just" hammering the threads allowed on the CPU's on the VM on where the `Local-AI` `Pod` is running.

Dang!

**Comment by mudler** [comment]
ok cool thanks @LarsBingBong ! - let's see if this fixes other issues related to missing instructions cpu set and collect more feedback. However I think your issue might be either related to the environment - can you open an issue with logs/info on the environment?

**Pull request merged**